cmd/restic/global.go

```diff
@@ -160,9 +162,14 @@ func restoreTerminal() {
 	}
 
 	AddCleanupHandler(func() error {
+		// only restore the terminal settings when they were modified
```

Comment:
I find this comment hard to understand easily. It talks about terminal settings being modified, but the flag that's checked is about whether or not password reading is active. They seem like two different things. For readability, can this comment be rephrased?

Reply:
I've adapted the comment to better match the following check. Right now the comment and the check afterwards yield the same result, but I agree that it was a bit confusing.